custom_components/hacs/__init__.py

```diff
@@ -135,7 +136,7 @@ async def async_startup():
             hass.config.path("custom_components/custom_updater.py"),
             hass.config.path("custom_components/custom_updater/__init__.py"),
         ):
-            if os.path.exists(location):
+            if await async_exists(location):
```

Comment:
Side note:
As this is a Python module and the check is to see if it's there or not, would this be a better approach?
```python
try:
  import custom_components.custom_updater
  HAS_CUSTOM_UPDATER = True
except ImportError:
  HAS_CUSTOM_UPDATER = False
```

Reply:
I think we don't know if another integration has been installed yet unless we (after) depend on it in the manifest. So I don't think we can use that approach. If it would have been a module that we know will be installed ahead of time, it would have been better.